### PR TITLE
"Serving Spark NLP via API" documentation

### PR DESCRIPTION
**Pull request by agsfer**
## Description

Adds "Spark NLP via API" articles to nlp.johnsnowlabs.com

## Motivation and Context

New articles have come up about serving Spark NLP via API, that have been shared in Medium and blog posts, and need to be available as documentation as well.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/_data/navigation.yml [hunk 1]
```diff
@@ -48,8 +48,12 @@ sparknlp:
         url:    /docs/en/display
       - title:  Experiment Tracking
         url:    /docs/en/mlflow
-      - title:  Productionizing Spark NLP
-        url:    /docs/en/production-readiness
+      - title:  Serving Spark NLP via API SynapseML
```
Comment (josejuanmartinez) [inline docs/_data/navigation.yml:51]: `:` after API

File: docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
+---
+layout: docs
+header: true
+seotitle: Spark NLP
+title: Serving Spark NLP via API Databricks and MLFlow
```
Comment (josejuanmartinez) [inline docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md:5]: `:` after API

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

File: docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md (hunk 2 above)
Comment (maziyarpanahi) [reply docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md:5]: @josejuanmartinez 
Can we make the names (also the URL path) more friendly? 

'Serving Spark NLP: MLFlow on Databricks`
'Serving Spark NLP: SynapseML`
'Serving Spark NLP: FastAPI`

File: docs/en/Serving_Spark_NLP_via_API_FastAPI.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 layout: docs
 header: true
 seotitle: Spark NLP
-title: Serving Spark NLP via API FastAPI
-permalink: /docs/en/spark_nlp_via_api_fastapi
+title: Serving Spark NLP&#58 FastAPI
```
Comment (josejuanmartinez) [inline docs/en/Serving_Spark_NLP_via_API_FastAPI.md:5]: Serving Spark NLP: FastAPI

File: docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 layout: docs
 header: true
 seotitle: Spark NLP
-title: Serving Spark NLP via API Databricks and MLFlow
+title: Serving Spark NLP&#58 MLFlow on Databricks
```
Comment (josejuanmartinez) [inline docs/en/Serving_Spark_NLP_via_API_Databricks_and_MLFlow.md:5]: Serving Spark NLP: MLFlow on Databricks

File: docs/_data/navigation.yml [hunk 5]
```diff
@@ -48,11 +48,11 @@ sparknlp:
         url:    /docs/en/display
       - title:  Experiment Tracking
         url:    /docs/en/mlflow
-      - title:  Serving Spark NLP via API SynapseML
+      - title:  Serving Spark NLP via API&#58 SynapseML
```
Comment (josejuanmartinez) [inline docs/_data/navigation.yml:51]: Serving Spark NLP: SynapseML

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

File: docs/_data/navigation.yml [hunk 6]
```diff
@@ -48,7 +48,7 @@ sparknlp:
         url:    /docs/en/display
       - title:  Experiment Tracking
         url:    /docs/en/mlflow
-      - title:  Serving Spark NLP via API&#58 SynapseML
+      - title:  Serving Spark NLP API&#58 SynapseML
```
Comment (josejuanmartinez) [inline docs/_data/navigation.yml:51]: Serving Spark NLP: SynapseML

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

**Review by josejuanmartinez — APPROVED** [review]

**Pull request merged**